src/developers/notary_server.md

```diff
@@ -0,0 +1,51 @@
+# Run a Notary Server
+
+This guide shows you how to run a [notary server](https://github.com/tlsnotary/tlsn/tree/dev/notary-server) in an Ubuntu server instance.
+
+## Configure Server Setting
+All the following setting can be configured in the [config file](https://github.com/tlsnotary/tlsn/blob/dev/notary-server/config/config.yaml).
+
+1. One *MUST* change the path of the following files, as by default dummy fixtures are used which is not secured at all
+
+    | File | Purpose | File Type | Compulsory to change |
+     ----- | ------- | ------------------ | -------------------- |
+    | TLS private key | Private key used for notary server's TLS certificate to establish TLS connection with prover | Compatible TLS private key in PEM format | Yes unless TLS is turned off |
+    | TLS certificate | Notary server's TLS certificate to establish TLS connection with prover | Compatible TLS certificate in PEM format | Yes unless TLS is turned off |
+    | Notary signature private key | Private key used for notary server's signature on the generated transcript of the TLS session with prover | A P256 elliptic curve private key in PEM format | Yes |
+    | Notary signature public key | Public key used for notary server's signature on the generated transcript of the TLS session with prover | A public key (in PEM format) that corresponds to the private key above | Yes |
```

Comment:
```suggestion
    | TLS private key | The private key used for the notary server's TLS certificate to establish TLS connections with provers | Compatible TLS private key in PEM format | Yes unless TLS is turned off |
    | TLS certificate | The notary server's TLS certificate to establish TLS connections with provers | Compatible TLS certificate in PEM format | Yes unless TLS is turned off |
    | Notary signature private key | The private key used for the notary server's signature on the generated transcript of the TLS sessions with provers | A P256 elliptic curve private key in PEM format | Yes |
    | Notary signature public key | The public key used for the notary server's signature on the generated transcript of the TLS sessions with provers | A matching public key (in PEM format) | Yes |
```

Reply:
What does "compatible" mean here?

Reply:
> What does "compatible" mean here?

Ah just wanted to mean it has to be key/cert that the TLS spec allows, will drop the term